docs/sources/operations/storage/boltdb-shipper.md

```diff
@@ -77,18 +81,21 @@ When running Loki in clustered mode there could be multiple ingesters serving wr
 *NOTE: To avoid any loss of index when Ingester crashes it is recommended to run Ingesters as statefulset(when using k8s) with a persistent storage for storing index files.*
```

Comment:
```suggestion
**Note:** To avoid any loss of index when Ingester crashes it is recommended to run Ingesters as statefulset(when using k8s) with a persistent storage for storing index files.
```